### PR TITLE
test(frameworks): change pytorch test using test_frameworks

### PR DESCRIPTION
**Pull request by larme**
## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by larme** [comment]
just kept data container part of `test_pytorch_impl.py`. I wonder where should we put them? @bojiang

File: .github/workflows/frameworks.yml [hunk 1]
```diff
@@ -120,6 +120,8 @@ jobs:
               - *runner
               - bentoml/fastai.py
               - bentoml/_internal/frameworks/fastai.py
+              - bentoml/_internal/frameworks/common/pytorch.py
+              - tests/integration/frameworks/test_frameworks.py
```
Comment (aarnphm) [inline .github/workflows/frameworks.yml:124]: fastai uses pytorch data container, hence it is needed to also track the changes from there

File: .github/workflows/frameworks.yml [hunk 2]
```diff
@@ -401,7 +404,7 @@ jobs:
           fi
 
       - name: Upload test coverage to Codecov
-        if: ${{ needs.diff.outputs.detectron2 == 'true' }}
+        if: ${{ needs.diff.outputs.fastai == 'true' }}
```
Comment (aarnphm) [inline .github/workflows/frameworks.yml:407]: fix: ci should track changes from fastai, not detectron :)

File: .github/workflows/frameworks.yml [hunk 3]
```diff
@@ -893,12 +896,22 @@ jobs:
...
+      - name: Run unit test for pytorch
+        shell: bash
+        run: |
+          OPTS=(--cov=bentoml --cov-config=./setup.cfg --cov-report=xml:"pytorch.unit.xml" --cov-report term-missing:skip-covered)
+          python -m pytest ./tests/integration/frameworks/test_pytorch_unit.py "${OPTS[@]}" || ERR=1
+          if [ $ERR -eq 1 ]; then
+            echo "unit tests for pytorch failed!"
+            exit 1
+          fi
+
```
Comment (aarnphm) [inline .github/workflows/frameworks.yml:908]: We will run the container tests here.

**Review by aarnphm — APPROVED** [review]
Run passed locally. Will wait for CI to pass.

**Pull request merged**